src/Console/Command/DescribeCommand.php

```diff
@@ -225,7 +225,13 @@ private function describeRule(OutputInterface $output, $name)
                 $old = $codeSample->getCode();
                 $tokens = Tokens::fromCode($old);
                 if ($fixer instanceof ConfigurableFixerInterface) {
-                    $fixer->configure($codeSample->getConfiguration());
+                    $configuration = $codeSample->getConfiguration();
+
+                    if (null === $configuration) {
```

Comment:
No, that just means the code sample uses the default configuration of the fixer.

The changes here prevent calling deprecated code in such case by passing an empty array instead of `null`, which is deprecated. We didn't notice the deprecated code usage before because that was not covered by tests yet.